src/lib.rs

```diff
@@ -17,12 +17,12 @@
 
 #![doc(html_root_url = "https://hyperium.github.io/mime.rs")]
 #![cfg_attr(test, deny(warnings))]
-#![cfg_attr(all(features = "nightly", test), feature(test))]
+#![cfg_attr(all(feature = "nightly", test), feature(test))]
 
 #[macro_use]
 extern crate log;
 
-#[cfg(features = "nightly")]
+#[cfg(feature = "nightly")]
```

Comment:
This should be exactly how it's working: on stable, without `--features nightly`, this cfg is false, and so the crate is not included.